test/demo/utils/hydrateNotification.ts

```diff
@@ -0,0 +1,31 @@
+import {NotificationOptions, Notification} from '../../../src/client';
+
+export function assertHydratedCorrectly(options: NotificationOptions, fullNote: Notification): void {
+    // Handle `id` and `date` seperately as these cannot be predicted if undefined
```

Comment:
We can't assert their exact values, but could still assert that they're not undefined? Or that they look like a provider-generated ID, and a valid Date object.

Reply:
:+1: